pycmr/granule.py

```diff
@@ -50,6 +51,64 @@ def point(self, point=None):
 
         return self
 
+    def downloadable(self, downloadable):
+        """
+        Set the downloadable value for the query.
+
+        Must be of type Boolean
+        """
+        if not isinstance(downloadable, bool):
```

Comment:
Two things here:

1. Raise exceptions instead of returning silently. Imagine the scenario where you make an invalid call like `api.downloadable("sup").execute()`. What would end up happening is the parameter never gets set, and the search sent would be a generic "give me all the CMR granules" search, regardless of their downloadable flag.
2. In order for chaining to work, all these parameter setting methods will need to `return self`